README.md

```diff
@@ -90,41 +95,41 @@ NewPipe supports multiple services. Our [docs](https://teamnewpipe.github.io/doc
+ 5. If you're interested in a specific feature or bugfix provided in a Pull Request in this repo, you can also download its APK from within the PR. Read the PR description for instructions. The great thing about PR-specific APKs is that they're installed side-by-side the official app, so you don't have to worry about losing your data or messing anything up.
 
-We recommend method 1 for most users. APKs installed using method 1 or 2 are compatible with each other, but not with those installed using method 3. This is due to the same signing key (ours) being used for 1 and 2, but a different signing key (F-Droid's) being used for 3. Building a debug APK using method 4 excludes a key entirely. Signing keys help ensure that a user isn't tricked into installing a malicious update to an app.
+We recommend method 1 for most users. APKs installed using method 1 or 2 are compatible with each other (meaning you can update your app using either method), but not with those installed using method 3. This is due to the same signing key (ours) being used for 1 and 2, but a different signing key (F-Droid's) being used for 3. Building a debug APK using method 4 excludes a key entirely. Signing keys help ensure that a user isn't tricked into installing a malicious update to an app. When using method 5, each APK is signed with a different random key supplied by GitHub Actions, so you cannot even update it. You will have to backup and restore the app data each time you wish to use a new APK.
 
 In the meanwhile, if you want to switch sources for some reason (e.g. NewPipe's core functionality breaks and F-Droid doesn't have the latest update yet), we recommend following this procedure:
 1. Back up your data via Settings > Content > Export Database so you keep your history, subscriptions, and playlists
 2. Uninstall NewPipe
 3. Download the APK from the new source and install it
 4. Import the data from step 1 via Settings > Content > Import Database
 
-## Contribution
-Whether you have ideas, translations, design changes, code cleaning, or real heavy code changes, help is always welcome.
-The more is done the better it gets!
+<b>Note: when you're importing a database into the official app, always make sure that it is the one you exported _from_ the official app. If you import a database exported from an APK other than the official app, it may break things. Such an action is unsupported, and you should only do so when you're absolutely certain you know what you're doing.</b>
 
-If you'd like to get involved, check our [contribution notes](.github/CONTRIBUTING.md).
+## Contribution
+Whether you have ideas, translations, design changes, code cleaning, or even major code changes, help is always welcome. The more is done the better it gets! If you'd like to get involved, check our [contribution notes](.github/CONTRIBUTING.md).
```

Comment:
```suggestion
Whether you have ideas, translations, design changes, code cleaning, or even major code changes, help is always welcome. The more getting done the better it gets! If you'd like to get involved, check our [contribution notes](.github/CONTRIBUTING.md).
```

Reply:
Well, that is also incorrect. Let's change it completely.

"The app gets better and better with each contribution, no matter how big or small!"

Reply:
What's wrong grammatically with "the more is done the betteer it gets"?

Reply:
Nothing, but it does read a bit weird. Let's replace it anyway.

Reply:
```suggestion
Whether you have ideas, translations, design changes, code cleaning, or even major code changes, help is always welcome. The app gets better and better with each contribution, no matter how big or small! If you'd like to get involved, check our [contribution notes](.github/CONTRIBUTING.md).
```